Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -1314,3 +1314,14 @@ function Add-AzureVMCustomScriptExtension
 
     Write-Verbose "Successfully added the custom script extension '$extensionName' for virtual machine '$vmName'"
 }
+
+function Check-ContainerNameAndArgs
+{
+    param([string]$containerName,
+          [string]$additionalArguments)
+    
+    if($containerName -eq '$root' -and ($additionalArguments -eq "/S" -or $additionalArguments -like "/S *" -or $additionalArguments -like "* /S *" -or $additionalArguments -like "* /S"))
```

Comment:
thanks @KrishnaAdityaB , The /S is case-insensitive. Will add a regex which checks for both the cases